<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnoozeTimesEdit</class>
 <widget class="QWidget" name="SnoozeTimesEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="upButton"/>
     </item>
     <item>
      <widget class="QToolButton" name="downButton"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="removeButton"/>
     </item>
     <item>
      <widget class="QToolButton" name="addButton"/>
     </item>
    </layout>
   </item>
  </layout>
  <action name="action_Add_Time">
   <property name="icon">
    <iconset theme="list-add" resource="remindmewidgets.qrc">
     <normaloff>:/icons/add.ico</normaloff>:/icons/add.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Add Time</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Ins</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
  </action>
  <action name="action_Remove_Time">
   <property name="icon">
    <iconset theme="list-remove" resource="remindmewidgets.qrc">
     <normaloff>:/icons/delete.ico</normaloff>:/icons/delete.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Remove Time</string>
   </property>
   <property name="shortcut">
    <string notr="true">Del</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
  </action>
  <action name="actionMove_Up">
   <property name="icon">
    <iconset theme="go-up" resource="remindmewidgets.qrc">
     <normaloff>:/icons/up.ico</normaloff>:/icons/up.ico</iconset>
   </property>
   <property name="text">
    <string>Move &amp;Up</string>
   </property>
   <property name="shortcut">
    <string notr="true">PgUp</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
  </action>
  <action name="actionMove_Down">
   <property name="icon">
    <iconset theme="go-down" resource="remindmewidgets.qrc">
     <normaloff>:/icons/down.ico</normaloff>:/icons/down.ico</iconset>
   </property>
   <property name="text">
    <string>Move &amp;Down</string>
   </property>
   <property name="shortcut">
    <string notr="true">PgDown</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="remindmewidgets.qrc"/>
 </resources>
 <connections/>
</ui>
